os/include/debug.h

```diff
@@ -78,6 +78,7 @@
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
+#define _alert lldbg
```

Comment:
1. Is this necessary instead of lldbg?
2. lldbg could cause log mixed. Could you confirm where you used this for log mixed?